domains/letum.json

```diff
@@ -5,6 +5,11 @@
       "email": ""
   },
   "record": {
-    "CNAME": "letum.pages.dev"    
+    "CNAME": "letum.pages.dev",
+    "MX": [
+      "mx1.improvmx.com.",
+      "mx2.improvmx.com."
+    ],
+    "TXT": "v=spf1 include:spf.improvmx.com ~all"
```

Comment:
Cannot have CNAME record in conjunction with other records.

Reply:
can you look into it again? i fixed the issue.